javatests/arcs/core/storage/handle/CollectionIntegrationTest.kt

```diff
@@ -168,21 +182,21 @@ class CollectionIntegrationTest {
 
     @Test
     fun addElementsWithTtls() = runBlockingTest {
-        val person = Person("John", 29, false)
+        val person = Person("John", 29.0, false)
         collectionA.store(person.toRawEntity())
         val creationTimestampA = collectionA.fetchAll().first().creationTimestamp
         assertThat(creationTimestampA).isNotEqualTo(RawEntity.UNINITIALIZED_TIMESTAMP)
         assertThat(collectionA.fetchAll().first().expirationTimestamp)
             .isEqualTo(RawEntity.UNINITIALIZED_TIMESTAMP)
 
-        val collectionC = CollectionImpl("collectionC", storageProxy, null, null, Ttl.Days(2), TimeImpl())
+        val collectionC = CollectionImpl("collectionC", storageProxy, null, Ttl.Days(2), TimeImpl(), schema=SCHEMA_A)
         storageProxy.registerHandle(collectionC)
         assertThat(collectionC.store(person.toRawEntity())).isTrue()
         val entityC = collectionC.fetchAll().first()
         assertThat(entityC.creationTimestamp).isGreaterThan(creationTimestampA)
         assertThat(entityC.expirationTimestamp).isGreaterThan(RawEntity.UNINITIALIZED_TIMESTAMP)
 
-        val collectionD = CollectionImpl("collectionD", storageProxy, null, null, Ttl.Minutes(1), TimeImpl())
+        val collectionD = CollectionImpl("collectionD", storageProxy, null, Ttl.Minutes(1), TimeImpl(), schema=SCHEMA_B)
```

Comment:
Nit: spaces between `schema`, the `=` and `SCHEMA_B`, also - looks like this might be too-long of a line.  If you're using AS/IJ, put your cursor on any arg, press alt+enter, then choose "Place arguments on separate lines"